package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "mattermost-webapp",
```

Comment:
Nit: I think this is unintentional and should be reversion. IIRC, it happens because some older versions of node 16 do not add this field.

Reply:
Which node version are you on? The change you have mentioned is really annoying for me; I have to revert it every time.

Reply:
npm version: `8.5.5`
node version: `v16.15.0`